core/services.py

```diff
@@ -1,5 +1,12 @@
+import logging
 from typing import Any
 
+from config.settings.base import APP_ENV
```

Comment:
```suggestion
from django.conf import settings
```

* this way of importing makes it overridable in tests
* it's clear in your code that it's a global setting you're accessing since it'll be `settings.APP_ENV`

Reply:
No longer needed as I have removed the IF statement using APP_ENV